dll/win32/browseui/shellbars/CBandSiteMenu.cpp

```diff
@@ -48,19 +48,6 @@ CBandSiteMenu::~CBandSiteMenu()
 
 HRESULT WINAPI CBandSiteMenu::FinalConstruct()
 {
-    HRESULT hr = SHGetFolderLocation(0, CSIDL_DESKTOP, NULL, 0, &m_DesktopPidl);
-    if (FAILED_UNEXPECTEDLY(hr))
-        return hr;
-
-    WCHAR buffer[MAX_PATH];
-    hr = SHGetFolderPathAndSubDirW(0, CSIDL_APPDATA | CSIDL_FLAG_CREATE, NULL, 0, L"Microsoft\\Internet Explorer\\Quick Launch", buffer);
-    if (FAILED_UNEXPECTEDLY(hr))
-        return hr;
-
-    m_QLaunchPidl.Attach(ILCreateFromPathW(buffer));
-    if (m_QLaunchPidl == NULL)
-        return E_FAIL;
-
```

Comment:
Please not that _GetMenuIdFromISFBand uses m_QLaunchPidl as well so it won't have the correct behaviour. Also, after it is initialized once, there is no need to free it again.